services/console/src/components/navbar/theme/ThemeToggle.astro

```diff
@@ -60,7 +60,6 @@ import { THEME_TOGGLE_ID, ThemeId } from "./theme";
 
   const initTheme = () => {
     const theme = getTheme();
-    setTheme(theme);
```

Comment:
Users who manually toggle the theme will still have their theme preference remain the same, regardless of system theme changes.